src/main/java/com/CezaryZal/manager/filters/validator/UserLoginValidatorService.java

```diff
@@ -0,0 +1,37 @@
+package com.CezaryZal.manager.filters.validator;
+
+import com.CezaryZal.entity.UserLogin;
+import com.CezaryZal.exceptions.IncorrectInput;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserLoginValidatorService {
+
+    private LoginFormValidator loginFormValidator;
+    private PasswordFormValidator passwordFormValidator;
+
+    public UserLoginValidatorService(LoginFormValidator loginFormValidator, PasswordFormValidator passwordFormValidator) {
+        this.loginFormValidator = loginFormValidator;
+        this.passwordFormValidator = passwordFormValidator;
+    }
+
+    public void isCorrectUserLogin(UserLogin userLogin) {
+        try {
+            throwIsEmptyUserLogin(userLogin);
+            loginFormValidator.validLogin(userLogin.getLogin());
+            passwordFormValidator.validPassword(userLogin.getPassword());
+        } catch (RuntimeException exc){
```

Comment:
Tak na pewno nie. Tworzysz z ogólnego exceptiona jakiś konkretny. Masz 100% pewności że ten RuntimeException będzie IncorrecIInputEXceptionem? Może sięzdarzyćże nie, to będzie ciężkie do zrozumienia z logów, co się tutaj zadziało

Reply:
Ok, poprawiłem